src/engine/source/kvdb2/include/kvdb2/kvdbHandlerCollection.hpp

```diff
@@ -0,0 +1,47 @@
+#ifndef _KVDBHANDLERCOLLECTION_H
```

Comment:
I think it should be _KVDB_HANDLER_COLLECTION_H

Reply:
Will be fixed in later documentation step.